### PR TITLE
Add models

### PR DESCRIPTION
**Pull request by cody-code-wy**
this is an update on PR #9

### CODE REVIEW

File: app/models/twitter_handle.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class TwitterHandle < ActiveRecord::Base
+
+  belongs_to :topic
+
+  has_many :tweets
+
+  validates :handle, presence: true
```
Comment (monicamow) [inline app/models/twitter_handle.rb]: the :handle field should be revised to :twitter_handle to remain consistent to the table name.

Comment (monicamow) [inline app/models/twitter_handle.rb]: we did not finish the validations...owner needs presence: true

Comment (cody-code-wy) [inline app/models/twitter_handle.rb]: @monicamow `owner` is nil for non spoof users. so validating presence is incorrect behaviour.

**Comment by cody-code-wy** [comment]
Accidentally pushed before closing.

## app/models/twitter_handle.rb

7: `+  validates :handle, presence: true`

Comments
=========

```
from monica:
the :handle field should be revised to :twitter_handle to remain consistent to the table name.
```

```
 from monica:
we did not finish the validations...owner needs presence: true
```

```
 from william:
@monicamow owner is nil for non spoof users. so validating presence is incorrect behaviour.
```

**Pull request closed without merging**